authentication/managing-security-context-constraints.adoc

```diff
@@ -6,15 +6,50 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+In {product-title}, you can use security context constraints (SCCs) to control permissions for the pods in your cluster.
+
+Default SCCs are created during installation and when you install some Operators or other components. As a cluster administrator, you can also create your own SCCs by using the OpenShift CLI (`oc`).
+
+[IMPORTANT]
+====
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues when some of the platform pods deploy or 
+ifndef::openshift-rosa[]
+{product-title} 
+endif::[]
+ifdef::openshift-rosa[]
+ROSA 
+endif::openshift-rosa[]
+is upgraded. Additionally, the default SCC values are reset to the defaults during some cluster upgrades, which discards all customizations to those SCCs.
+ifdef::openshift-origin,openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-rosa[]
+
+Instead of modifying the default SCCs, create and modify your own SCCs as needed. For detailed steps, see xref:../authentication/managing-security-context-constraints.adoc#security-context-constraints-creating_configuring-internal-oauth[Creating security context constraints].
+endif::[]
+====
+
+ifdef::openshift-dedicated[]
+[NOTE]
+====
+In {product-title} deployments, you can create your own SCCs only for clusters that use the Customer Cloud Subscription (CCS) model. You cannot create SCCs for {product-title} clusters that use a Red Hat cloud account, because SCC resource creation requires `cluster-admin` privileges.
```

Comment:
> Customer Cloud Subscription (CCS)
> Red Hat cloud account

@yuwang-RH @xueli181114 I have reviewed common parts (i.e. those applying to all OCP/ROSA/OSD). But other parts, like the contents referenced here are not OCP scope, I've no more background. Need you to review these. Thanks!